src/timepicker/reducer/timepicker.actions.ts

```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { Action } from '../../mini-ngrx/index';
+import { Action } from '../../mini-ngrx';
```

Comment:
core files should use import from `index` files, otherwise, it will not work for system.js folks